ddtrace/tracer.py

```diff
@@ -7,6 +7,7 @@
 from .sampler import AllSampler
 from .writer import AgentWriter
 from .span import Span
+from .settings import PP_KEY
```

Comment:
instead of doing that, I'd suggest to remove `PP_KEY` from the previous file and:
```python
from .constants import PROCESSING_PIPELINE_KEY as PP_KEY
```